string name
bool emergency
int32 battery 
geometry_msgs/Point current_position
string current_state
geometry_msgs/Point target_position